Cargo.toml

```diff
@@ -36,6 +36,9 @@ serde_derive = "1.0.116"
 cidr-utils = "0.5.0"
 itertools = "0.9.0"
 trust-dns-resolver = { version = "0.19.5", features = ["dns-over-rustls"] }
+anyhow = "1.0.32"
+subprocess = "0.2.6"
+text_placeholder = { version = "0.3", features = ["struct_context"] }
```

Comment:
Now we have a version `0.4` that uses less memory 😄

Reply:
Nice! I bumped it. :)